src/modules/plusgpl/filter_rotoscoping.c

```diff
@@ -327,7 +327,7 @@ static int filter_get_image(mlt_frame frame,
     mlt_properties frame_properties = MLT_FRAME_PROPERTIES(frame);
     int b_width = mlt_properties_get_int(frame_properties, "meta.media.width");
     int b_height = mlt_properties_get_int(frame_properties, "meta.media.height");
-    if (b_height == 0) {
+    if (b_width == 0 || b_height == 0) {
```

Comment:
I was not confident about this change, but b_width and b_height are actually not used for anything.

Reply:
Done